Sofa/Component/Visual/src/sofa/component/visual/LineAxis.cpp

```diff
@@ -38,9 +40,10 @@ void registerLineAxis(sofa::core::ObjectFactory* factory)
 using namespace sofa::defaulttype;
 
 LineAxis::LineAxis()
-    : d_axis(initData(&d_axis, std::string("xyz"),  "axis", "Axis to draw"))
-    , d_size(initData(&d_size, 10.f,  "size", "Size of the squared grid"))
-    , d_thickness(initData(&d_thickness, 1.f,  "thickness", "Thickness of the lines in the grid"))
+    : d_axis(initData(&d_axis, std::string("xyz"),  "axis", "Axis to draw."))
+    , d_size(initData(&d_size, 10.f,  "size", "Size of the lines. Give a negative value for infinite lines."))
```

Comment:
Are you suggesting that infinite lines have negative length ? Isn't that mathematically shocking  ?

Reply:
Should I change for the following: 
```
    , d_size(initData(&d_size, 10.f,  "size", "Size of the lines (segments)."))
    , d_infinite(initData(&d_infinite, false,  "infinite", "If true, ignore the 'size' and draw an infinite line."))
```

Reply:
Hello, 


To me it is better, I let @hugtalbot and other handle it. 
Maybe it is possible to fill the size with +infinity and have the widget show that "value" with +infinity.